bin/setup

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
```

Comment:
Should this script have `set -e` (exits if any subcommand fails), like the other scripts?